db/seeds.rb

```diff
@@ -8,7 +8,21 @@
 
 Admin.create! first_name: 'Sam', last_name: 'Admin', email: 'sam@admin.com', password: 'password'
 
-role = MemberRole.create! role: "Role"
+# Create basic roles
+president_role = MemberRole.create! role: "President"
+internal_role = MemberRole.create! role: "Internal VP"
+external_role = MemberRole.create! role: "External VP"
+tech_role = MemberRole.create! role: "VP of Technology"
+# projects_role = MemberRole.create! role: "VP of Projects"
+leader_role = MemberRole.create! role: "Project Leader"
+developer_role = MemberRole.create! role: "Project Developer"
+# alumni_relations_role = MemberRole.create! role: "Alumni Relations Director"
+community_role = MemberRole.create! role: "Community Outreach Director"
+# intersocietal_role = MemberRole.create! role: "Intersocietal Relations Director"
+media_role = MemberRole.create! role: "Media Director"
+# corporate_role = MemberRole.create! role: "Corporate Relations Director"
```

Comment:
Let's just uncomment these to create all the roles so we don't have to do it later.